presto-raptor/src/test/java/com/facebook/presto/raptor/RaptorQueryRunner.java

```diff
@@ -57,7 +57,12 @@ public static DistributedQueryRunner createRaptorQueryRunner(
             Map<String, String> extraRaptorProperties)
             throws Exception
     {
-        DistributedQueryRunner queryRunner = new DistributedQueryRunner(createSession("tpch"), 2, extraProperties);
+        DistributedQueryRunner queryRunner = DistributedQueryRunner.builder(createSession("tpch"))
+                .setNodeCount(2)
+                // TODO: enable failure-detector for raptor
```

Comment:
Maybe also elaborate like you did for `JmxQueryRunner`?

Reply:
sure.  I didn't look into exactly why it was failing (some errors about trying to move things around during start up), but I'll add a note that it's not enabled yet because of failures.